_data-prepper/pipelines/configuration/processors/truncate.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: default
+title: truncate
+parent: Processors
+grand_parent: Pipelines
+nav_order: 121
+---
+
+# truncate
+
+The `truncate` processor truncates a key's value at the beginning, the end or, on both sides of the value string based on processor's configuration. If the key's value is a list, then each member in the string list is truncated. Non-string members of the list are left untouched. When the `truncate_when` option is provided, the truncation of the input is done only when the condition specified is true for the event being processed.
+
+## Configuration
+
+You can configure the `truncate` processor using the following options.
+
+Option | Required | Type | Description
+:--- | :--- | :--- | :---
+`entries` | Yes | String list | A list of entries to add to an event.
+`source_keys` | No | String list | The list of sources keys that will be modified by the processor. The default value is an empty list which indicates truncating all values.
+`truncate_when` | No | Conditional expression | A condition that, when met, determines when the truncate operation is performed. 
+`start_at` | No | Integer | Where inside the string value to start the truncation process. Default is `0`, which means the truncation of each key's value starts at the beginning.
+`length` | No | Integer| The length of the string after truncation. When not specified, the processor will measure the length based on where the string ends.
+
+For the `truncate` processor to run either the `start_at` or `length` options must be present in the configuration. For greater customization, you can define both values inside the configuration. 
```

Comment:
"For greater customization" is a bit vague (Customization of what? Why?). Can this sentence be rephrased as "You can define both values in the configuration in order to [what specifically the user can do by defining both values]."